app/models/user.rb

```diff
@@ -28,6 +28,17 @@ class User < ApplicationRecord
   validate :lat_lng_unchanged
   validates_length_of :name, minimum: 3, maximum: 30
 
+  # Reverse geocoding for city finder
+  reverse_geocoded_by :lat, :lng do |user, results|
+    if geo = results.first
+      user.city = geo.city
+      user.country_code = geo.country_code
+    end
+  rescue => e
+    Rails.logger.error("Failed to geocode city or country_code due to #{e.message}")
```

Comment:
@nronas Can we log and report the exception(AppSignal)?
Analysing logs and implementing exception reports from there is an overkill for this app, so just directly reporting is probably better. 
We have couple of helpers to do that i think the mix is called ExceptionHelpers. 
ex: log_and_report(e)

Reply:
Also, you can pass any context params to help debug the issue quicker.

Reply:
Had no clue that we use AppSignal. will amend

Reply:
@nronas yeah, my fault. Didn't mention i anywhere, may be we should have a conventions guide somewhere

Reply:
no harm done